src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+
+    private final List<Number> matchNumber;
+    private final Number bonusNumber;
+
+    public WinningLotto(List<Number> matchNumber, Number bonusNumber) {
+        this.matchNumber = matchNumber;
+        this.bonusNumber = bonusNumber;
+    }
+
+    public boolean isExistBonusNumber(List<Number> lottoNumbers) {
+        return Collections.frequency(lottoNumbers, bonusNumber) > 0 ? true : false;
+    }
```

Comment:
6개의 로또 번호 중 보너스 번호는 오직 1개만 존재하거나, 존재하지 않는다는 규칙이 있는데요.
frequency 대신 contains를 사용해보는건 어떨까요? :)

```suggestion
    public boolean isExistBonusNumber(List<Number> lottoNumbers) {
        return lottoNumbers.contains(bonusNumber);
    }
```